src/modules/jobs.rs

```diff
@@ -0,0 +1,25 @@
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Creates a segment to show if there are any active jobs running
+pub fn segment<'a>(context: &'a Context) -> Option<Module<'a>> {
```

Comment:
It looks like unit tests are failing because of this:
```suggestion
pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
```

I would take the time to review our continuous integration.